internal/maputil/maputil.go

```diff
@@ -0,0 +1,11 @@
+package maputil
```

Comment:
CI is failing because of that:

```suggestion
//go:build go1.22
package maputil
```